db/migrate/20170217184205_add_power_state_property_to_physical_server.rb

```diff
@@ -0,0 +1,5 @@
+class AddPowerStatePropertyToPhysicalServer < ActiveRecord::Migration[5.0]
+  def change
+    add_column :physical_servers, :power_state, :string
```

Comment:
My only concern is that in the other tables we have separated the concept of "raw_power_state" and "power state".  "raw_power_state" is the raw string that we get from the provider, whereas "power_state" is a normalized value, mainly used for reporting. cc @blomquisg 

I'll merge anyway, but you might need a followup PR to add "raw_power_state".  Also, please be sure that usage of the power state column is limited to the regular normalized values (like what we show in the quadicons in the UI.